condacolab.py

```diff
@@ -89,11 +89,38 @@ def install_from_url(
         stderr=STDOUT,
         text=True,
     )
+
+    # installing google.colab packages, matplortlib-base and psutil.
```

Comment:
```suggestion
    # installing google.colab packages, matplotlib-base and psutil.
```

Reply:
I would also add _why_ we need this. In order to launch a Python kernel, the Colab server expects certain packages to be installed; these ones.